usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -325,13 +324,8 @@ def when_non_zero_award_spending(self, query):
             total_outlay=self.outlay_field_annotation, total_obligation=self.obligated_field_annotation
         ).exclude(total_outlay=0, total_obligation=0)
 
-    @property
-    def unique_file_c_awards(self):
-        delimiter = Value("|")
-        return ConcatAll("piid", delimiter, "parent_award_id", delimiter, "fain", delimiter, "uri")
-
     def unique_file_c_award_count(self):
-        return Count(self.unique_file_c_awards, distinct=True)
+        return Count("distinct_award_key", distinct=True)
```

Comment:
Again pleased but maybe surprised that this is the only Django query clause that needed refactoring. All that trouble with the `award/amount` endpoint, and this is the line that was doing it?

Actually, isn't there obligation and outlay sums that also utilize this "award_identifer" concatenation? These values that aren't being populated:
![image](https://user-images.githubusercontent.com/14953413/92008639-6d2d4180-ed15-11ea-94c0-dd0696a6952a.png)

Reply:
Your surprise is surprising to me. It's a function, define once and reuse is the goal, right?

Counting File C awards was in more endpoints, but the award count was changed to be counts of File D awards.